src/index.js

```diff
@@ -74,10 +77,14 @@ export default function ({ types: t, traverse: _traverse }) {
         const { parentPath } = path;
         const explicitReturn = parentPath.isReturnStatement();
         const implicitReturn = parentPath.isArrowFunctionExpression();
-
-        const openingElement = elementOpenCall(path.get("openingElement"), this);
-        const closingElement = elementCloseCall(path.get("openingElement"), this);
-        const children = buildChildren(path.get("children"), this);
+        const openingElementPath = path.get("openingElement");
+
+        const openingElement = elementOpenCall(openingElementPath, this);
+        const closingElement = elementCloseCall(openingElementPath, this);
+        const isSkipping = hasSkip(openingElementPath.get("attributes"), this) ||
+          isComponent(openingElementPath.get("name"), this);
```

Comment:
This falls under React's [Containment](https://facebook.github.io/react/docs/composition-vs-inheritance.html#containment) principle. But, since iDOM doesn't really have a way to instantiate a component with state yet, we can leave this out for now.